docs/DataFeeds.md

```diff
@@ -1,5 +1,8 @@
 # Data Feeds
 ## KLASS
+Data is imported as JSON from https://data.ssb.no/api/klass/v1, (documented here: https://data.ssb.no/api/klass/v1/api-guide.html).
```

Comment:
Remove this first link https://data.ssb.no/api/klass/v1 as it will always return 404. Link to the api-guide is enough.